portfolio/src/main/webapp/contact.html

```diff
@@ -15,19 +15,35 @@
       <nav class="w3-bar w3-black">
         <a href="index.html" class="w3-bar-item w3-button">Home</a>
         <a href="experience.html" class="w3-bar-item w3-button">Experience</a>
-        <a href="/" class="w3-bar-item w3-button">Interest</a>
+        <a href="interest.html" class="w3-bar-item w3-button">Interest</a>
         <!-- TODO: To be implemented -->
         <a href="#" class="w3-bar-item w3-button">Gallery</a>
-        <a href="contact.html" class="w3-bar-item w3-button">Contact Me</a>
+        <a href="/" class="w3-bar-item w3-button">Contact Me</a>
         <a class="w3-bar-item w3-right"><b>Rania Rusly</b></a>
       </nav>
 
-      <div class="w3-cell-row section">
+      <div class="section">
         <h1>Comments</h1>
         <button onclick=getComments()>apa kabar?</button>
         <div id="comments"></div>
       </div>
 
+      <div class="section">
+        <form action="/text" method="POST">
+
+          <p>Enter a comma-separated list of words:</p>
+          <textarea name="text-input">one, two, three</textarea>
+          <br/><br/>
+
+          <p>Choose some options:</p>
+          <input type="checkbox" name="upper-case" value="true"> Upper-case
+          <br/>
+          <input type="checkbox" name="sort" value="true"> Alphabetize
+          <br/><br/>
+
+          <input type="submit" />
+        </form>
+      </div>
```

Comment:
This seems like the next step after fetching the comments from the server, can you please have it in a new pull request instead?